tools/scripts/generate_docker_release_matrix.py

```diff
@@ -73,9 +74,15 @@ def main() -> None:
         choices=["nightly", "test", "release", "all"],
         default=os.getenv("CHANNEL", "nightly"),
     )
+    parser.add_argument(
+    "--generate_dockerhub_images",
+    help="Whether to generate Docker Hub images (default: False)",
+    action="store_true",
+    default=False,  
```

Comment:
Since we are setting this as env var, this should be:
```
type=str,
default=os.getenv("GENERATE_DOCKERHUB_IMAGES", "false"),
```
Here is an example: https://github.com/pytorch/test-infra/blob/main/tools/scripts/generate_binary_build_matrix.py#L633